src/App.js

```diff
@@ -6,11 +6,15 @@ import { Switch, Route } from "react-router-dom";
 import Home from "./pages/WelcomePage";
 import MyCakes from "./pages/MyCakes";
 import Favs from "./pages/Favs";
+import SingleCakePage from "./pages/SingleCakePage";
 
 function App() {
   return (
     <div className="App">
       <Switch>
+        <Route path="/cakes/:id">
+          <SingleCakePage />
+        </Route>
         <Route component={Favs} path="/favs" />
```

Comment:
Why are you following this way in this Route instead of doing it like the one you did before (nesting the component inside the Route component)

Reply:
I noticed it while researching and tried to test it here. Forgot to remove it. Will keep the Route consistent by sticking to one approach. :)